data/autoyast_sle12/create_hdd/create_hdd_ha_sles.xml.ep

```diff
@@ -305,21 +305,16 @@
         <products config:type="list">
             <product><%= uc $get_var->('SLE_PRODUCT') %></product>
         </products>
-        <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
         <packages config:type="list">
-        % foreach (values %$addons) {
+            <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
+            % foreach (values %$addons) {
             <package><%= lc($_->{name}) %>-release</package>
-        % }
+            % }
+            <!-- end of workaround -->
         </packages>
-        <!-- end of workaround -->
     </software>
     <services-manager>
-        % if ($check_var->('DESKTOP', 'gnome')) {
```

Comment:
I don't know why remove these lines, I prefer the profile should support both 'gnome' and 'textmode'.

Reply:
The test suite we are replacing is only in text mode, we neve install gnome. Of course it is not a big deal to keep it, but I tried to remove anything we don't need to make the profile as simple as possible.

Reply:
Ok, then go ahead. I just think a different way from you, I want to make the profile can support more test to avoid other effort to modify or create profile for other requirement. :)

Reply:
In Yast job group we've been working in this manner for the past 2-3 years: avoid conditions in code when possible, and instead make tests as specific as possible. It makes code more readable and maintainable, but of course we have more files in the end. We chose to do that because we were spending hours trying to understand bugs in tests, due to too many conditions in code, like "if is_sle 12+ BUT not sle15 OR ARCH=s390x" etc...